content/en/logs/log_collection/javascript.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Browser Log Collection
+kind: documentation
+aliases:
+  - /logs/log_collection/web_browser
+further_reading:
+- link: "logs/processing"
+  tag: "Documentation"
+  text: "Learn how to process your logs"
+- link: "logs/processing/parsing"
+  tag: "Documentation"
+  text: "Learn more about parsing"
+- link: "logs/explorer"
+  tag: "Documentation"
+  text: "Learn how to explore your logs"
+- link: "logs/explorer/analytics"
+  tag: "Documentation"
+  text: "Perform Log Analytics"
+- link: "logs/faq/log-collection-troubleshooting-guide"
+  tag: "FAQ"
+  text: "Log Collection Troubleshooting Guide"
+---
+
+
+
+Send logs to Datadog from web browsers or other Javascript client thanks to the Datadog Client-side JavaScript logging library.
+
+With the `datadog-logs` library you are able to send log directory to Datadog from JS clients and leverage the following features:
+
+* Use the library as a logger. Everything is forwarded to Datadog as JSON documents.
+* Add `context` and extra attributes.
+* Forward every JavaScript errors.
+* Forward JavaScript's console logs.
+* Track real client IP address and useragent.
+* Automatic bulk posts.
+
+## Get the Public API Key
+
+**Public API Keys are in Private Beta** 
+
+For security reasons, [API keys][2] cannot be used as they would be exposed in the Javascript code. To collect Logs from web browsers, a [Public API keys][3] must be used.
+To manage your Public API keys, go to your [Datadog API configuration page][4] in the `Public API Key` section as shown here:
+
+{{< img src="logs/log_collection/public_key.png" style="width:80%;" alt="Public API Keys" responsive="true" >}}
+
+Read our [Public API keys documentation][3] to learn more about the restrictions that applies.
+
+## Configure the Javascript logger
+
+The following parameters can be used to configure the library to send logs to Datadog:
```

Comment:
this is my newby showing, but we should state where it is that you update these parameters

Reply:
The examples should be right below.